### PR TITLE
LOOP-1025 Jhbate/sprint14

### PR DESCRIPTION
**Pull request by jh-bate**
see https://tidepool.atlassian.net/browse/LOOP-1025. FYI I mimicked the Dash setup

### CODE REVIEW

File: MockKit/MockPumpManager.swift [hunk 1]
```diff
@@ -15,19 +15,23 @@ public protocol MockPumpManagerStateObserver {
...
+    public var failureReason: String? {
         switch self {
         case .pumpSuspended:
             return "Pump is suspended"
         case .communicationFailure:
             return "Unable to communicate with pump"
         case .bolusInProgress:
             return "Bolus in progress"
+        case .missingSettings:
+            return "Missing Settings."
```
Comment (darinkrauss) [inline MockKit/MockPumpManager.swift]: [nit] Might want to remove the trailing `.` to be consistent with the other failure reasons here.

File: MockKitUI/MockPumpManager+UI.swift [hunk 2]
```diff
@@ -16,21 +16,21 @@ extension MockPumpManager: PumpManagerUI {
     public static func setupViewController() -> (UIViewController & CompletionNotifying & PumpManagerSetupViewController) {
         return MockPumpManagerSetupViewController.instantiateFromStoryboard()
     }
-
+
```
Comment (darinkrauss) [inline MockKitUI/MockPumpManager+UI.swift]: [nit] Might want to undo these whitespace-only changes.

**Review by darinkrauss — APPROVED** [review]
A couple of real small nits, but LGTM! 🚢 

Also, might want to run it by Michael or Pete so they are aware.

**Review by mpangburn — APPROVED** [review]
LGTM! Just to clarify, when you say this mimics the Dash behavior—that implementation will allow you to continue through the UI with a missing delivery limit setting?
(I ask because that issue seems fixable, in which case you wouldn't need to have a test for this in the first place.)

**Comment by jh-bate** [comment]
@mpangburn the dash setup (and now also the simulator setup) will come up with an alert when you miss one of the delivery limit settings and hence cannot setup loop into a valid state

**Pull request merged**